packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -276,6 +276,19 @@ abstract class FlutterCommand extends Command<void> {
       help: 'Build a release version of your app${defaultToRelease ? ' (default mode)' : ''}.');
   }
 
+  void addShrinkingFlag() {
+    argParser.addFlag('shrinking',
+      negatable: true,
+      defaultsTo: true,
+      help: 'Whether to enable shrinking on release mode.'
```

Comment:
nit: Should we specifically call out R8 here?  That's probably more discoverable via searching than just shrinking.

Reply:
Renamed as "shrink" since it could be used on other platforms as well.